.github/workflows/release.yaml

```diff
@@ -28,6 +28,8 @@ jobs:
           for tag in ${tags[@]}; do
               oras push ghcr.io/aquasecurity/trivy-policies:${tag} \
               --config /dev/null:application/vnd.cncf.openpolicyagent.config.v1+json \
+              --annotation "org.opencontainers.image.source=$GITHUB_SERVER_URL/$GITHUB_REPOSITORY" \
+              --annotation "org.opencontainers.image.revision=$GITHUB_SHA" \
```

Comment:
What are the values for these variables? Are they documented somewhere?

Reply:
The annotation semantics are documented at https://github.com/opencontainers/image-spec/blob/v1.1.0/annotations.md#pre-defined-annotation-keys

The environment variables are documented at https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables